src/main.rs

```diff
@@ -197,6 +203,10 @@ fn build_hyperfine_options<'a>(
         (None, None) => {}
     };
 
+    options.setup_command = matches
+        .values_of("setup")
+        .map(|values| values.map(String::from).collect::<Vec<String>>());
```

Comment:
Once you remove `.multiple(true)`, this can be written in analogy to che `cleanup_command` line below, i.e. with `value_of` instead of `values_of`.